lib/nostrum/gateway/message_delete_event.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Nostrum.Gateway.MessageDeleteEvent do
+  @moduledoc """
+  Struct representing a Message Delete Event.
+  """
+
+  alias Nostrum.Snowflake
+
+  defstruct [
+    :id,
+    :channel_id,
+    :guild_id
+  ]
+
+  @typedoc "Id of the deleted message."
```

Comment:
We typically don't use periods at the end of our typedocs.

Reply:
Resolved in 283604a.